app/models/user.rb

```diff
@@ -33,7 +33,7 @@ def owned_by(user)
   # :confirmable, :lockable, :timeoutable and :omniauthable,
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :timeoutable,
-         :omniauthable, :omniauth_providers => [:google_oauth2]
+         :omniauthable, :omniauth_providers => [:google, :google_billing]
```

Comment:
can you explain why we use 'google' instead of google_oauth2 now?

Reply:
If you look at this: https://github.com/broadinstitute/single_cell_portal_core/blob/jb-pet-service-accounts/config/initializers/devise.rb#L257

What we're doing is configuring two different Omniauth providers, both of which are `google_oauth2` (via the `strategy_class: OmniAuth::Strategies::GoogleOauth2` declaration).  But by using these two different "names", we can have different scope requests that map to discrete callback endpoints.  You can think of it like a `scope` or `namespace` declaration in your `routes.rb` file - it's just a way for Devise & Omniauth to know which provider/strategy to load for each auth endpoint.

This is the only way that you can have separate scope lists unless you do some really complicated dynamic configs via a setup callback endpoint or a fancy setup proc like [this](https://gist.github.com/jiggneshhgohel/4a79aa26cb628533fe132295cffc45b2).  That seemed like a higher risk than this approach entailed.

Reply:
gotcha, I knew we needed the new  name to handle the billing-included scopes, wasn't sure why we had to rename the 'default' scope to 'google' from google_oauth2.  thanks for the explanation